packages/smooth_app/ios/Runner/lol.lproj/InfoPlist.strings

```diff
@@ -1 +1 @@
-"NSCameraUsageDescription" = "crwdns192692:0crwdne192692:0";
+"NSCameraUsageDescription" = "crwdns192868:0crwdne192868:0";
```

Comment:
Uhm

Reply:
@M123-dev that's the lol fake locale, that's to be expected